docs/framework/network-programming/how-to-send-data-using-the-webrequest-class.md

```diff
@@ -152,6 +142,8 @@ The following procedure describes the steps used to send data to a server. This
   
 ## Example  
   
+The following code example shows how to send data to a web server and read the data in its response:  
```

Comment:
- Close the unnecessary space between the method name and the opening parenthesis.
- Since this is a complete example, rather than presenting it inline, it's best to include it in the dotnet/samples repo.